cep-0099.md

```diff
@@ -0,0 +1,77 @@
+# CEP 99 - Changes to Conda Deprecation Schedule
+
+<table>
+<tr><td> Title </td><td> Changes to the Conda Deprecation Schedule </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Daniel Holth &lt;dholth@anaconda.com&gt;</td></tr>
+<tr><td> Created </td><td> Dec 16, 2025</td></tr>
+<tr><td> Updated </td><td> Dec 16, 2025</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Defines portions of the conda codebase that are not subject to the deprecation
+schedule, demarcated by following Python library conventions.
+
+## Motivation
+
+[CEP 9](https://github.com/conda/ceps/blob/main/cep-0009.md) describes a
+deprecation schedule to warn about removals from the codebase but does not
+describe any part of conda's codebase that is exempt from the deprecation
+schedule. This 10-15 month deprecation cycle, applied to all symbols in the
+codebase, is an unnecessary barrier to change.
+
+At the time it was not unreasonable to assume that any part of conda could have
+been used as a library by unknown dependents, but this is not a good assumption
+for new code. Instead, most new code is not called from outside conda and should
+be allowed to change as needed between conda releases.
+
+By applying Python type-checker API conventions to codebases that fall under
+CEP 9, this proposal aims to create a place for experiments and implementation
+details in conda.
+
+## Specification
+
+This CEP proposes that conda adopt the [Python Library
+Interface](https://typing.python.org/en/latest/spec/distributing.html#library-interface-public-and-private-symbols)
+convention for new code. In this convention underscore-prefixed symbols are
+considered private and are not part of the library's interface but `__all__`
+overrides these rules. Symbols may be re-exported into the public interface with
+forms like `import X as X`. Python type checkers understand these conventions
+and can let developers know if they are using conda's public interface.
+
+Newly added private code would not be subject to the deprecation policy. This
+code could change as necessary instead of after a deprecation cycle.
+
+Private portions of newly-added public modules, i.e. underscore-prefixed symbols
```

Comment:
Is there a convention for when (and at what level) we might want to introduce `__all__` into a module or not?

Reply:
I would audit existing code especially projects under https://github.com/conda, add those, and also provide deliberate API exports for new code but not necessarily worry about `__all__` when conda imports from itself.